backend/clj/rehearser/api/exercise.clj

```diff
@@ -15,38 +40,50 @@
     {:status 404
      :body "Not found, not changed"}))
 
+(defn response-update-one [result]
```

Comment:
The update mechanism was a bit of a mess, could update this in `main` separately.